app/models/client.rb

```diff
@@ -0,0 +1,34 @@
+require './app/models/parser'
```

Comment:
your models are automatically required in every file with this line that lives in your `environment.rb` file:

```ruby
# require the model(s)
Dir[APP_ROOT.join('app', 'models', '*.rb')].each { |file| require file }
```

no need to also require it manually here